frontend/src/api/backendClient.ts

```diff
@@ -7,6 +7,12 @@ export interface User {
   aboutMe?: string
 }
 
+// export interface UpdatedUser {
+//   firstName?: string
+//   lastName?: string
+//   aboutMe?: string
+// }
```

Comment:
I think you already messaged about needing to delete this, but just commenting for completeness